Ish2and3

cknuth opened the pull request:
Fixed issue #2 so that the homepage in packagist now points to the CAOS wiki.

Fixed issue #3 so that the source in packagist now points to the github repository.

cknuth commented:
Let he who may merge the pull request come unto us and say with great prominence, "Let it be so!".

@ahoffmann-wi

hamdrew commented:
For some reason there are merge conflicts yet I see none in the diff. Can you try rebasing master on your branch and pushing again? Kind of weird.

nblair commented:
I'm wondering if e0263fb is the issue?

hamdrew commented:
Oh, yes. It looks like you merged in changes from Bitbucket, @cknuth. A rebase should fix this. However, it might behoove you to wipe out your local repo and clone fresh from github.

cknuth commented:
Ah it looks like I had rebased on the bitbucket master branch and not the github master. Should be fixed now

Ha looks like we figured that out at the same time. I've rebased on the github master branch now.

hamdrew commented:
And on this day, he said to His children, "the Merge is good," and they rejoiced.

He also said:

fixes #2
fixes #3  

and they continued to rejoice, a little confused.
